benchmarks/htap/__init__.py

```diff
@@ -4,13 +4,17 @@
 def add_parser(subparsers):
     parser = subparsers.add_parser('htap')
     parser.add_argument(
-        '--oltp-workers', default=1, type=int, help=(
+        '--oltp-workers', default=32, type=int, help=(
         'The number of OLTP workers executing TPC-C transactions (i.e. simulated clients).'))
 
     parser.add_argument(
         '--olap-workers', default=1, type=int, help=(
         'The number of OLAP workers running TPC-H-like queries.'))
 
+    parser.add_argument(
+        '--target-tps', default=None, type=int, help=(
```

Comment:
Don't we need `required=True` in some of these? Or is it the default for all arguments that `add_argument`?

Reply:
no this is not the default. if it is not specified the default is put in which is good enough.